Runtime/Utilities/WebRequestRest/Response.cs

```diff
@@ -42,5 +42,10 @@ public Response(bool successful, string responseBody, byte[] responseData, long
             ResponseData = responseData;
             ResponseCode = responseCode;
         }
+
+        /// <summary>
+        /// Failure response type
+        /// </summary>
+        public static Response Failure => new Response(false, "", null, -1);
```

Comment:
Changing to string.empty is good.  However it is easier to test for a code < 0 rather than a specific error code, So I probably wouldn't change that as it is subjective. This failure is used as a general failure and not a specific one.